client/components/command-pallette/README.md

```diff
@@ -0,0 +1,51 @@
+# Command Palette
+
+The Command Palette is a React component used to display a text input used to search and filter a list of commands. It can be accessed in the Calypso using the shortcut `cmd+k`.
+
+The command palette is inspired by [Core](https://github.com/WordPress/gutenberg/blob/trunk/packages/commands/README.md) implementation used in the Gutenberg editor.
+
+## Add a command
+
+In order to add a new command to the palette, you can add it in `useCommandsArrayWpcom` or create a new hook and include it to `useCommandPallette` hooks.
```

Comment:
Couple of questions here:

1. Can we provide guidance on "best practices" for adding a command? e.g. What makes a good command vs. not, what language should be used to describe the command, etc.?
2. Should we have all WordPress.com commands registered via a central mechanism, so it's easy to keep tabs on what commands are added? I think that developers adding commands shouldn't have to worry about mounting the `<WpcomCommandPalette />` component.

Reply:
> Can we provide guidance on "best practices" for adding a command? e.g. What makes a good command vs. not, what language should be used to describe the command, etc.?

Added here https://github.com/Automattic/wp-calypso/tree/add/command-palette-add-readme/client/components/command-pallette#best-practices-for-defining-a-command

> Should we have all WordPress.com commands registered via a central mechanism, so it's easy to keep tabs on what commands are added? I think that developers adding commands shouldn't have to worry about mounting the <WpcomCommandPalette /> component.

I was more explicit that we already are mounting `WpcomCommandPalette` in "all" Claypso pages for WPcom. So developers do not need to mount it again, just add a command into the array.